### ISSUE TITLE
Avatars may be blocked by browser plugins

### ISSUE THREAD

**Issue by stefanb** [WeblateOrg/weblate#412]
Gravatar is blocked by default (or in a strict setting, not sure) by popular [Ghostery](http://www.ghostery.com/) browser addon. Other providers might become blocked at any later time. It would be nice to have some local "generic person" image ready to handle such cases and be shown instead (probably as css backround of the avatar image).
